src/components/Layout/Layout.js

```diff
@@ -27,15 +27,6 @@ export function Layout({
   const isMobile = useIsMobile();
   const [isOpen, setIsOpen] = React.useState(false);
   const toggleIsOpen = () => setIsOpen((prev) => !prev);
-  React.useEffect(() => {
-    if (isOpen) {
-      window.scrollTo({
-        top: 0,
-        left: 0,
-        behavior: 'smooth',
-      });
-    }
-  }, [isOpen]);
```

Comment:
Think another option which might be a better fix would be to disable the `scrollTo` when `isMobile`?

Reply:
This is not a mobile issue. It's only happening on desktop. (for me at least)

Reply:
Yeah that's what I was getting at - missed an `!`, I meant when `!isMobile`. If this code is useful on mobile but not needed on desktop, we could change it so it doesn't run on desktop?

Reply:
Same here, only happens at desktop

Reply:
Simply disabling scrollTo on desktop won't work either unfortunately. If it's just flatout disabled the site would not scroll up when you do a fast click on a link and would want it to scroll up. That would be a regression in functionality.